.github/workflows/merge_to_main_production.yaml

```diff
@@ -10,6 +10,8 @@ on:
 env:
   PRIVATE_ECR: ${{ secrets.PRODUCTION_AWS_ACCOUNT }}.dkr.ecr.ca-central-1.amazonaws.com/notify
   API_LAMBDA_IMAGE: api-lambda:a4e4b54
+  HEARTBEAT_LAMBDA_IMAGE: heartbeat:fb6dcdfda3c8e21ef997e75d1cb872bb08916cf3
+  SYSTEM_STATUS_LAMBDA_IMAGE: system_status:fb6dcdfda3c8e21ef997e75d1cb872bb08916cf3
```

Comment:
We're setting these manually on first iteration and automatically later?

Reply:
yes